examples/bert/run_pretraining.py

```diff
@@ -291,6 +301,53 @@ def train_step(self, data):
         self.nsp_accuracy.update_state(data["next_sentence_labels"], nsp_preds)
         return {m.name: m.result() for m in self.metrics}
 
+class AdamLRSchedule(keras.optimizers.schedules.LearningRateSchedule):
```

Comment:
rename LinearDecayWithWarmup

Reply:
Yeah, it's better to name it this way. Would also help reduce the extra comments in the docstring.